### DISCUSSION TITLE
If-Then Function

### DISCUSSION THREAD

**Discussion by sbrsquest2** [KL-Psychological-Methodology/ESMira#14] [Questions] [answered]
Hello everyone,

I am very new to using ESMira and am currently trying to set up a questionnaire on the website. I have come across the problem that I would like to insert an if-then condition. So if you choose answer A, you have to answer another question, if you choose answer B you don't and can simply continue the questionnaire without the follow-up question. Unfortunately, I don't know exactly how to do this and haven't found anything about it yet. Does anyone have any ideas? That would be great.

**Comment by SelinaDev** [comment, marked as answer]
Hi! Thank you for your question! Unfortunately, ESMira currently does not have such a feature. We also see the necessity, and this is relatively high up on our list of things to work on. However, I cannot yet give any estimates when this feature might become available.

**Reply by sbrsquest2** [reply]
Hi, ok, thank you very much for your fast reply.
